packages/paste-core/components/ai-chat-log/stories/composer.stories.tsx

```diff
@@ -229,12 +224,27 @@ export const AIChatLogComposer = (): React.ReactNode => {
           ariaLabel="Message"
           placeholder="Type here..."
           onChange={handleComposerChange}
+          editorInstanceRef={editorRef}
         >
           <ClearEditorPlugin />
-          <SendButtonPlugin onClick={submitMessage} />
           <EnterKeySubmitPlugin onKeyDown={submitMessage} />
         </ChatComposer>
-      </Box>
+        <ChatComposerActionGroup>
+          <Button variant="secondary_icon" size="reset">
+            <AttachIcon decorative={false} title="Attach" />
```

Comment:
suggestion(non-blocking): something a little more descriptive for the icons might be helpful in the examples here (e.g. "attach a file to your message")